mpas_analysis/ocean/ocean_regional_profiles.py

```diff
@@ -331,6 +360,11 @@ def run_task(self):
                                     'data': np.arange(nVertLevels)})
 
         vertMask = vertIndex < dsRestart.maxLevelCell
+        if self.max_bottom_depth:
```

Comment:
```suggestion
        if self.max_bottom_depth is not None:
```
I believe this is the preferred way to check for `None`.  Treating it as a boolean can have unintended consequences like `0.0` getting treated as `False`.